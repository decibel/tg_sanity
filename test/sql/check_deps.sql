\set ECHO none

\set QUIET true

BEGIN;
CREATE EXTENSION pgtap;
\echo EXPECT: required extension "tg_sanity" is not installed
CREATE EXTENSION tg_sanity_tap;
ROLLBACK;
BEGIN;
CREATE EXTENSION tg_sanity;
\echo EXPECT: required extension "pgtap" is not installed
CREATE EXTENSION tg_sanity_tap;
ROLLBACK;

-- vi: expandtab sw=2 ts=2
