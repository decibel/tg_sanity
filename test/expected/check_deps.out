\set ECHO none
EXPECT: required extension "tg_sanity" is not installed
ERROR:  required extension "tg_sanity" is not installed
EXPECT: required extension "pgtap" is not installed
ERROR:  required extension "pgtap" is not installed
